server/src/cores/storage.core.ts

```diff
@@ -50,8 +50,10 @@ export class StorageCore {
     private cryptoRepository: ICryptoRepository,
     private repository: IStorageRepository,
     systemConfigRepository: ISystemConfigRepository,
+    @Inject(ILoggerRepository) private logger: ILoggerRepository,
```

Comment:
You can't and don't want to have the inject there

Reply:
it seems to me that we wanna inject the new logger .
ref: https://github.com/immich-app/immich/pull/8823